app.py

```diff
@@ -151,6 +161,79 @@
 
 DICT_TEMP_RES = {}
 
+DICT_BANNER_INFO = {"banner_message_info": {}}
+
+# Update banner info with the message in .env
+@app.context_processor
+def display_info():
+    global DICT_BANNER_INFO
+
+    try:
+        env_banner_info = dotenv_values(".env")["BANNER_INFO"]
```

Comment:
Can't load any "BANNER_INFO" variable in a `.env`file. Why not keeping config variables in the original `config.py` file ?